templates/db.inifile.params.j2

```diff
@@ -104,7 +104,7 @@ NW_GetMasterPassword.masterPwd = {{ master_password }}
 # NW_HDB_DB.javaSchemaPassword =
 
 # Install the SAP HANA database client in a central or local directory. Possible values are: 'SAPCPE', 'LOCAL'
-{%- if product_id_header in ['S4HANA1809', 'S4HANA1909'] %}
+{%- if 'S4HANA' in product_id_header %}
```

Comment:
This comparisons were done to avoid the usage of `S4HANA1709` in there, as the installation failed in this version.
Keep this in mind, as we don't want new installations updates breaking older version